### PR TITLE
Update to Vanilla 3.0

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Update Sass
- Update Vanilla version and import
- Deleted `pattern_tiles` and other scss classes that weren't being used
- Updated variables to new names
- Fixed broken nav 
- Added `u-hide--medium` class to the video on the homepage to fix it filling the whole width of the screen on medium screen sizes

## QA

- View site at: 
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the site for anything breaking. Check particularly the nav and everything at different screen sizes

### CODE REVIEW

File: static/sass/_blockquote.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // These are the blockquote rules from the Vanilla Dev branch at commit 19a80ec
 // https://github.com/vanilla-framework/vanilla-framework/blob/develop/scss/_base_blockquotes.scss
 // When it lands in main and the depency is updated in this repo this can be removed
-@import 'vanilla-framework/scss/settings';
+@import "vanilla-framework/scss/settings";
```
Comment (bartaz) [inline static/sass/_blockquote.scss:4]: Judging from the comment on top of this file needed at all?
The commit it mentions is very old, and the blockquote styles in Vanilla look almost identical (apart from some small spacing differences). It should use the official Vanilla styles ideally.

File: static/sass/_sidebar.scss [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@use 'sass:math';
+
 .docs-container {
```
Comment (bartaz) [inline static/sass/_sidebar.scss:3]: Something to verify - is this needed anywhere on the site?

It seems the docs section (https://microk8s.io/docs) has been using official Vanilla docs layout, so this styling maybe is not used anywhere anymore?

File: static/sass/_blockquote.scss (hunk 1 above)
Comment (bethcollins92) [reply static/sass/_blockquote.scss]: Good point, removed both the files

File: templates/contact-us.html [hunk 3]
```diff
@@ -97,9 +97,9 @@ <h3 class="p-card__title">Any questions? Call us</h3>
         <h3 class="p-card__title">Need help with Ubuntu?</h3>
         <p>If you are just looking for some free support for yourself, you can try:</p>
         <ul class="p-list--divided">
-          <li class="p-list__item"><a class="p-link--external" href="https://askubuntu.com/">Ask Ubuntu</a></li>
-          <li class="p-list__item"><a class="p-link--external" href="https://ubuntuforums.org/">The Ubuntu forum</a></li>
-          <li class="p-list__item"><a class="p-link--external" href="https://help.ubuntu.com/">Help docs</a></li>
+          <li class="p-list__item"><a  href="https://askubuntu.com/">Ask Ubuntu</a></li>
+          <li class="p-list__item"><a  href="https://ubuntuforums.org/">The Ubuntu forum</a></li>
+          <li class="p-list__item"><a  href="https://help.ubuntu.com/">Help docs</a></li>
```
Comment (carkod) [inline templates/contact-us.html:102]: Additional space between `a` and `href`

File: templates/docs/document.html [hunk 4]
```diff
@@ -71,9 +71,9 @@ <h1 class="u-no-margin--bottom">{{ document.title }}</h1>
 
 			<div class="p-strip is-shallow">
 				<div class="p-notification--information">
-					<p class="p-notification__response">
-						Last updated {{ document.updated }}. <a href="{{ forum_url }}{{ document.topic_path }}">Help improve this document in the forum</a>.
-					</p>
+					<div class="p-notification__content">
+						<p class="p-notification__message">Last updated {{ document.updated }}. <a href="{{ forum_url }}{{ document.topic_path }}">Help improve this document in the forum</a>.</p>
+          </div>
```
Comment (carkod) [inline templates/docs/document.html:76]: This indenting looks a bit weird?

Comment (minkyngkm) [reply templates/docs/document.html:76]: It looks okay to me!

**Review by minkyngkm — APPROVED** [review]
It's really minor but I'm seeing two spaces after `a` tag in many places. Other than that, it looks all good to me! Nice work! @bethcollins92 🎉

**Comment by bethcollins92** [comment]
Updated, thanks @minkyngkm

**Pull request merged**